cms/djangoapps/contentstore/utils.py

```diff
@@ -295,9 +295,10 @@ def get_group_display_name(user_partitions, xblock_display_name):
         group name (String): Group name of the matching group.
     """
     for user_partition in user_partitions:
-        for group in user_partition['groups']:
-            if str(group['id']) in xblock_display_name:
-                return group['name']
+        if user_partition['scheme'] == 'random':
```

Comment:
You can avoid having to know about the scheme name by using the helper method _get_split_user_partitions_ in split_test_module.py.

for user_partition in get_split_user_partitions(user_partitions)